src/client/debug.html

```diff
@@ -11,10 +11,10 @@
     <meta property="og:site_name" content="WebGME"/>
     <meta property="og:description" content="Web-based Generic Modeling Environment"/>
 
-    <!--<script>-->
-        <!--document.write('<script src="http://' + (location.host || 'localhost').split(':')[0] +-->
-                <!--':35729/livereload.js?snipver=1"></' + 'script>')-->
-    <!--</script>-->
+    <script>
```

Comment:
is it ok if it becomes 'default'?
I though it need some configuring with the livereload